teamsupport/models.py

```diff
@@ -1,19 +1,31 @@
 from property_caching import cached_property
 from querylist import QueryList
 
+from teamsupport.services import TeamSupportService
 from teamsupport.errors import MissingArgumentError
 
 
+TICKET_STATUS_NEW = '212203'
+TICKET_TYPE_SUPPORT = '35731'
```

Comment:
Configuration? I can't agree with this. I don't think this can change ever. They are just IDs from `TicketStatus` table and `TicketType` table.